jerry-core/ecma/operations/ecma-container-object.c

```diff
@@ -721,6 +721,31 @@ ecma_op_container_set_weak (ecma_object_t *const key_p, /**< key object */
   ecma_collection_push_back (refs_p, container_value);
 } /* ecma_op_container_set_weak */
 
+/**
+ * Helper method for the Map.prototype.set and Set.prototype.add methods to swap the sign of the given value if needed
+ *
+ * See also:
+ *          ECMA-262 v6, 23.2.3.1 step 6
+ *          ECMA-262 v6, 23.1.3.9 step 6
+ *
+ * @return ecma value
+ */
+static ecma_value_t
+ecma_op_container_set_swap_sign_helper (ecma_value_t this_arg) /*< this arg */
```

Comment:
Name is a bit awkward as the implementation does not swap the sign, but just nozmalize-s negative zero tto positve zero.
Maybe `ecma_op_container_set_noramlize_zero` ?